cli/tsc/mod.rs

```diff
@@ -402,7 +402,10 @@ impl State {
 }
 
 fn normalize_specifier(specifier: &str) -> Result<ModuleSpecifier, AnyError> {
-  resolve_url_or_path_deprecated(specifier).map_err(|err| err.into())
+  // TODO(bartlomieju): ideally we shouldn't need to call `current_dir()` on each
+  // call - maybe it should be caller's responsibility to pass it as an arg?
+  let cwd = std::env::current_dir().context("Unable to get CWD")?;
+  resolve_url_or_path(specifier, &cwd).map_err(|err| err.into())
```

Comment:
Maybe `resolve_url_or_path` should receive a function for the cwd since it's not always used internally?

Reply:
I was thinking about it, but decided not to do it - it should force you to be more careful and think if you really need to resolve a path.